stl/inc/concepts

```diff
@@ -122,26 +122,26 @@ namespace ranges {
         struct _Cpo {
             template <class _Ty1, class _Ty2>
                 requires _Use_ADL_swap<_Ty1, _Ty2>
-            constexpr void operator()(_Ty1&& __t, _Ty2&& __u) const
+            constexpr _STATIC_CALL_OPERATOR void operator()(_Ty1&& __t, _Ty2&& __u) _CONST_CALL_OPERATOR

```

Comment:
`constexpr _STATIC_CALL_OPERATOR` is the opposite of our conventional order. Occurs 3 times in this file.